.cirrus.yml

```diff
@@ -0,0 +1,44 @@
+task:
+  only_if: $CIRRUS_BRANCH == 'main' || $CIRRUS_BRANCH == 'develop' || $CIRRUS_TAG != '' || $CIRRUS_PR != ''
+  env:
+    CIRRUS_CLONE_SUBMODULES: true
+
+  matrix:
+    - name: Test build
+
+      arm_container:
+        matrix:
+          - image: python:3.7
+          - image: python:3.11
+
+      install_build_dependencies_script: |
```

Comment:
Ironically these jobs are failing now, so I'll probably have to debug why locally later.

Reply:
https://cirrus-ci.com/github/scikit-hep/fastjet/ci%2Fadd-cirrus-ci